cmake/Modules/VolkPython.cmake

```diff
@@ -36,11 +36,12 @@ if(PYTHON_EXECUTABLE)
 else(PYTHON_EXECUTABLE)
 
     #use the built-in find script
+    set(Python_ADDITIONAL_VERSIONS 3.4 3.5 3.6)
     find_package(PythonInterp 2)
 
     #and if that fails use the find program routine
     if(NOT PYTHONINTERP_FOUND)
-        find_program(PYTHON_EXECUTABLE NAMES python python2 python2.7 python2.6 python2.5)
+        find_program(PYTHON_EXECUTABLE NAMES python python2 python2.7 python2.6 python3)
```

Comment:
In these cases, the parens are syntactically irrelevant in py2. I added the `print_function` import in the python files not because they were particularly necessary, but because it will now throw an error if someone tries to use the older style print statement. We could add the `print_function` import into the cmake file snippet but it seems a bit much for a 1-liner.

Let me know and I'll update my pull request tomorrow. Thanks for the review.